check-status.sh

```
@@ -0,0 +1,58 @@
+cd frontend
+yarn audit --level moderate 2>&1 | grep "vulnerabilities found" | sed 's/^/     /'
+cd ..
+
+echo ""
+echo "✅ 4. BUILDS"
+echo "   Frontend build artifacts:" $(ls frontend/dist/ 2>/dev/null | wc -l) "files"
+echo "   Python environment:" $(python -c "import fastapi; print('✅ Working')" 2>/dev/null || echo "❌ Issues")
+
+echo ""
+echo "📊 5. CODE QUALITY (INFORMATIONAL)"
+echo "   Python linting (ruff):" $(ruff check src/ tests/ 2>/dev/null && echo "✅ Clean" || echo "⚠️ Issues")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)